dashboard.py

```diff
@@ -67,8 +64,10 @@ def start(self, root, started):
         if not self.observed and not polling:
             self.observed = self.observer.schedule(self, self.currentdir)
 
-        if __debug__:
-            print('%s Dashboard "%s"' % (polling and 'Polling' or 'Monitoring', self.currentdir))
+        if polling:
+            logger.debug(f'Polling Dashboard "{self.currentdir}"')
+        else:
+            logger.debug(f'Monitoring Dashboard "{self.currentdir}"')
```

Comment:
Do you want these explicitly to be double quoted? using `!r` would quote them and also immediately indicate if something isnt a string